README.md

```diff
@@ -259,7 +273,6 @@ Class | Method | HTTP request | Description
  - [TagListResponseTags](docs/Model/TagListResponseTags.md)
  - [TagServers](docs/Model/TagServers.md)
  - [TemplitizeStorageRequest](docs/Model/TemplitizeStorageRequest.md)
- - [Timezone](docs/Model/Timezone.md)
  - [TimezoneListResponse](docs/Model/TimezoneListResponse.md)
  - [TimezoneListResponseTimezones](docs/Model/TimezoneListResponseTimezones.md)
  - [Zone](docs/Model/Zone.md)
```

Comment:
After
## issues 
There should be a `# Development` part of this ReadMe so that OSS guys can participate in the development. There should be sub sections like "Install" and "Testing"